src/Driver/AbstractMySQLDriver.php

```diff
@@ -52,7 +53,7 @@ public function getDatabasePlatform(ServerVersionProvider $versionProvider): Abs
         return new MySQLPlatform();
     }
 
-    public function getExceptionConverter(): ExceptionConverter
+    public function getExceptionConverter(): ExceptionConverterInterface
```

Comment:
This one was not narrowed in https://github.com/doctrine/dbal/pull/4926, it's just as narrow on 3.x… I think it's fine not to document that change as breaking since `ExceptionConverter` is internal.

Reply:
No, the type is wider on 3.x.

https://github.com/doctrine/dbal/blob/2caad211fc51713d7e803545d603270d9fbb8d41/src/Driver/AbstractMySQLDriver.php#L200-L203

I'm restoring the 3.x return type declaration.